core/ui/src/views/SoftwareCenterAppInstances.vue

```diff
@@ -323,6 +323,17 @@
                       })
                     }}
                   </div>
+                  <template v-if="app.docs.relnotes_url">
+                    <span>
+                      <cv-link
+                        class="row icon-and-text"
+                        :href="app.docs.relnotes_url"
+                        target="_blank"
+                      >
+                        {{ $t("common.release_notes") }} <Launch20 />
```

Comment:
Since we have an icon too, for uniformity I'd change the cv-link with a NsButton with kind="ghost"